Core/src/game/functions/region/sites/group/SitesGroup.java

```diff
@@ -47,6 +50,9 @@ public final class SitesGroup extends BaseRegionFunction
 	
 	/** Direction chosen. */
 	private final DirectionsFunction dirnChoice;
+	
+	/** The condition */
+	private final BooleanFunction isVisibleFn;
```

Comment:
this new member needs to be accounted for down below in methods like gameFlags() and the concepts ones and other methods